server/service/individualServices/LinkAnalysis/PrepareProvideTransmitterStatusOfParallelLinksForwarding.js

```diff
@@ -0,0 +1,347 @@
+     *   Prepare request and send to 
+     *  RequestForProvidingTransmitterStatusOfParallelLinksCausesReadingTransmitterStatusFromMwdi.Response
+     *     provideTransmitterStatusOfParallelLinksRequestBody#requestor-receive-operation
+     *****************************************************************************************************/
+    let traceIndicatorIncrementer = 1;
+    let requestBody = {
+        requestId: requestId
+    }
+    if (parallelLinkListResponse) {
+        traceIndicatorIncrementer = parallelLinkListResponse.traceIndicatorIncrementer;
+        if (parallelLinkListResponse.parallelLinkList) {
+            requestBody.parallelLinkList = parallelLinkListResponse.parallelLinkList;
+        }
+    }
+    requestBody = await onfFormatter.modifyJsonObjectKeysToKebabCase(requestBody);
+    EventDispatcherWithResponse.BuildAndTriggerRestRequestToRequestor(
+        requestorProtocol,
+        requestorAddress,
+        requestorPort,
+        requestorReceiveOperation,
+        requestHeaders,
+        requestBody,
+        "POST",
+        traceIndicatorIncrementer++);
+}
+
+/**
+ * Prepare requestBody for callback 
+ *  RequestForProvidingTransmitterStatusOfParallelLinksCausesReadingTransmitterStatusFromMwdi.Response by addressing callbacks
+ *  RequestForProvidingTransmitterStatusOfParallelLinksCausesReadingTransmitterStatusFromMwdi.DetermineLinkEndpoints
+ *  RequestForProvidingTransmitterStatusOfParallelLinksCausesReadingTransmitterStatusFromMwdi.GetAirInterfaceConfigFromCache
+ *  RequestForProvidingTransmitterStatusOfParallelLinksCausesReadingTransmitterStatusFromMwdi.GetAirInterfaceStatusFromLive
+ * @param {List}  parallelLinks parallel links list 
+ * @param {Object}  requestHeaders Holds information of the requestHeaders like Xcorrelator , CustomerJourney,User etc.
+ * @param {Integer} traceIndicatorIncrementer traceIndicatorIncrementer to increment the trace indicator 
+ * @returns parallelLinkListResponse contains parallel-link-list and traceIndicatorIncrementer
+ */
+async function formulateRequestBodyforResponse(parallelLinks, requestHeaders, traceIndicatorIncrementer) {
+    let parallelLinkList = [];
+    let parallelLinkListResponse = {};
+    if (parallelLinks && parallelLinks.length !== 0) {
+        for (let i = 0; i < parallelLinks.length; i++) {
+            let parallelLinkDetails = {};
+            let transmitterStatusList = [];
+            let linkId = parallelLinks[i];
+            if (parallelLinks.length > 1) {
+
+                /****************************************************************************************
+                 *  Fetching linkEndPointsDetails mountName , uuid , localId 
+                 ****************************************************************************************/
+
+                let linkEndPointsDetailsList = await determineLinkEndPoints(linkId, requestHeaders, traceIndicatorIncrementer++);
```

Comment:
as discussed, since we have same implementation in linkRelatedSwitchingoperation.js, we shall transfer this function to prepareForwardingAutomation.js file